sbc_compassion/models/correspondence.py

```diff
@@ -102,7 +102,7 @@ class Correspondence(models.Model):
     page_ids = fields.One2many(
         'correspondence.page', 'correspondence_id')
     nbr_pages = fields.Integer(
-        string='Number of pages', compute='_compute_nbr_pages')
+        string=_('Number of pages'), compute='_compute_nbr_pages')
```

Comment:
Normally the strings inside fields are automatically added in translation files without needing to use the translation function. This is only for the code part